docs/tutorials/01_electronic_structure.ipynb

```diff
@@ -190,7 +201,7 @@
    "source": [
     "es_problem = ElectronicStructureProblem(driver)\n",
     "second_q_op = es_problem.second_q_ops()\n",
-    "print(second_q_op[0])"
+    "print(second_q_op[\"ElectronicEnergy\"])"
```

Comment:
In theory we have the `main_property_name` on the problem that avoids knowing/hard-coding a string value explicitly here. Should we be using that, or at least showing that here. I know the main goal was to go to the dictionary, but we had that way to allow users to get the main property without having to know. Here for instance we could get that name, print it as well as printing the operator.

ParticleNumber usage below evidently cannot be altered.

Reply:
I figured that this is really just a temporary fix until the 0.5 release. At that point `main_property_name` won't exist any more. So maybe we can keep it simple like here?